Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM clux/muslrust:latest as builder
+
+# Cache dependencies
+RUN mkdir ogn-web-gateway
+WORKDIR /ogn-web-gateway
+RUN mkdir src; echo "fn main(){}" > src/main.rs
```

Comment:
why are you creating a dummy file here?

Reply:
Oh, yeah this is a bit of a bummer, rust doesn't have a way to just download deps, the idea is that if cargo.toml does not change, the deps wont have to be pulled again. See https://github.com/rust-lang/cargo/issues/2644. The only thing is with just a `touch` of main.rs the build will exit non-zero.